web/Gemfile

```diff
@@ -0,0 +1,11 @@
+source "https://rubygems.org"
+
+gem "nanoc"
+gem "webrick"
+gem "asciidoctor"
+gem "sass"
+gem "w3c_validators"
+
+group 'nanoc' do
+  gem "nanoc-live"
+end
```

Comment:
Looking at this you could set the env var `BUNDLE_WITHOUT: nanoc` in the workflow file to not install this. Not insisting, but just as a hint.

Reply:
Sure, done.